### ISSUE TITLE
Carthage fails because of missing file

### ISSUE THREAD

**Issue by jodyheavener** [watson-developer-cloud/ios-sdk#363]
When I `carthage update --platform iOS` it builds everything until **RelationshipExtractionV1Beta**, where it fails because of the following:

```
The following build commands failed:
    CopyPlistFile /Library/Developer/Xcode/DerivedData/WatsonDeveloperCloud-bnlrrrdfudasexfivpmkoicqhcen/Build/Products/Release-iphoneos/RelationshipExtractionV1BetaTests.xctest/Credentials.plist Source/SupportingFiles/Credentials.plist
(1 failure)
```

~~Sure enough, it looks like Credentials.plist is [missing](https://github.com/watson-developer-cloud/ios-sdk/tree/master/Source/SupportingFiles).~~ I now see that `before_install` in the Travis file creates the file, but alas I still get this error!

**Comment by glennrfisher** [comment]
Thanks for letting us know! Sorry you ran into this problem.

I just pushed an update to fix this problem with [v0.4.2](https://github.com/watson-developer-cloud/ios-sdk/releases/tag/v0.4.2). If you run `carthage update --platform iOS` again then all frameworks should build successfully.

For what it's worth, the `Credentials.plist` file is a file that we use internally to store our team's credentials to testing services. It's only included in test targets (which shouldn't be built by Carthage). There was a bug with the `RelationshipExtractionV1Beta` scheme, though, that caused Carthage to try and build the `RelationshipExtractionV1BetaTests` target. Of course, that build failed because it requires the `Credentials.plist` file which we do not commit to the repository.

**Comment by glennrfisher** [comment]
As far as I can tell v0.4.2 fixes this problem. I'm going to close the issue, but please let us know if you run into any bugs. We'll do our best to fix them quickly.
